File: P5-NascarSeasonWebAPI/NascarPage/NascarPage/DTOs/LecturaPistaDTO.cs
using System.ComponentModel.DataAnnotations;

namespace NascarPage.DTOs
{
    public class LecturaPistaDTO
    {
        public int Id { get; set; }
        public string Nombre { get; set; } = null!;
        public string Distancia { get; set; } = null!;
        public string FotoPrimaria { get; set; } = null!;
        public string FotoSecundaria { get; set; } = null!;
        public string FotoTerciaria { get; set; } = null!;
        public int Vueltas { get; set; }
        public int? Orden { get; set; }
        public bool Disputada { get; set; }
        public bool EnElCalendario { get; set; }
    }
}

File: P6-NascarAngularPage/src/app/componentes/calendario/LecturaPistaDTO.ts
export interface LecturaPistaDTO {
    id: number;
    nombre: string;
    distancia: string;
    fotoPrimaria: string;
    fotoSecundaria: string;
    fotoTerciaria: string;
    vueltas: number;
    orden: number | null;
    disputada: boolean;
    enElCalendario: boolean;
}
export interface PatchPistaDTO {
    nombre: string;
    distancia: string;
    vueltas: number | null | undefined;
    orden: number | null | undefined;
    disputada: boolean;
    enElCalendario: boolean;
}

export interface CrearPistaDTO extends PatchPistaDTO {
    fotoPrimaria: File | null | undefined;
    fotoSecundaria: File | null | undefined;
    fotoTerciaria: File | null | undefined;
}